EditableTable/styles/RenderStyles.tsx

```diff
@@ -9,7 +9,7 @@ import { detailsHeaderStyles } from './DetailsListStyles';
 export const _onRenderDetailsHeader: IDetailsListProps['onRenderDetailsHeader'] = props => {
   if (props) {
     props.checkboxVisibility = CheckboxVisibility.always;
-    return <DetailsHeader {...props} styles={detailsHeaderStyles} />;
+    return <DetailsHeader {...props} className='columnsHeader' styles={detailsHeaderStyles} />;
```

Comment:
Can't we add the columnsHeader styles into the detailsHeaderStyles?

Reply:
no, we can't. DetailsHeaderStyles does not provide these style properties.